AnnotationTool/AnnotationTool/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for AnnotationTool project.
+
+Generated by 'django-admin startproject' using Django 4.2.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-lih(kwz5ku7ler4=*29)p=wgnpb%71h7c@b)(9^8=_-r)1q@pz'
```

Comment:
Just like the comment says, `SECURITY WARNING`!
Move the variable out of the code and keep it inside your environment variables.